drivers/dma/Kconfig.mcux_edma

```diff
@@ -6,7 +6,7 @@
 config DMA_MCUX_EDMA
 	bool "Enable MCUX DMA driver"
 	depends on HAS_MCUX_EDMA
-	select NOCACHE_MEMORY if HAS_MCUX_CACHE
+	imply NOCACHE_MEMORY if HAS_MCUX_CACHE
```

Comment:
the NOCACHE_MEMORY better not be disablable by users, in RT series without this, user need control the cache by them self, do you sure you want this?

Reply:
Problem here is, that the Kinetis KE1xF series has `CONFIG_HAS_MCUX_CACHE=y` but `CONFIG_ARCH_HAS_NOCACHE_MEMORY_SUPPORT=n`, so the dependency would not be fulfilled by the `select` statement.

The `imply` statement is like `select` except that it honours dependencies. Thus for SoCs with `CONFIG_HAS_MCUX_CACHE=y` that fulfils the dependencies for `CONFIG_NOCACHE_MEMORY=y` the result will be the same as with the `select` statement.